docs/stories/demo.md

```diff
@@ -0,0 +1,85 @@
+# 10101 Demo
+
+We decided to publish a separate demo post so [the pitch](https://makers.bolt.fun/story/10101-pitch--441) can stay concise and focus on being a pitch 😉
+In this demo post that showcases what we have built and how we got there so you get a better idea of how we work and our values.
+
+## We did it! - The demo
+
+10101 is a mobile Lightning wallet, that offers non-custodial trading services.
+The app-demo showcases how we integrate multiple services, with CFD trading being the first service that we implement.
+We did a public demo where we showcase the app live, [here's the recording](https://rumble.com/v1wr04i-10101-demo.html).
+Below the most important steps of the demo are showcased as gifs.
+
+As with every self-sovereign Lightning wallet upon startup the wallet will prompt the user to make sure to create a wallet backup:
+
+TODO: GIF
+
+In order to start using the wallet the user must be in possession of Bitcoin.
+The users needs to deposit Bitcoin into the wallet and open an initial channel to be part of the network.
+We step the user through this process one action at the time.
+
+First the user is prompted to deposit Bitcoin:
+
+TODO: GIF
+
+The user sends funds to the wallet's Bitcoin address.
+Once the wallet picked up the funding the user is prompted to open a channel with the `10101` maker:
+
+TODO: GIF
+
+The user's Lightning node running on the mobile phone opens a channel with the maker.
+The maker doubles the amount the user puts in, so that both sides are funded in preparation for using non-custodial trading protocols.
+In the future we see a P2P marketplace here, where `10101` acts as an enabler to connect (automated) market-makers and app-users.
+
+Once a channel was established the user can open a CFD position over the channel with the `10101` maker:
+
+TODO: GIF
+
+The user can close the position into the channel distributing the fund according to the profit/loss of the CFD position:
+
+TODO: GIF
+
+## How did we get there? - A bit of history
+
+A bit more than a year ago we started working on [ItchySats](https://www.itchysats.network/), a non-custodial CFD trading solution built using DLCs for the self-sovereign Bitcoiner.
+ItchySats was shipped on Umberl, raspiblitz, Citadel, MyNode and Start9 - but we never had the feeling that we were driving adoption enough.
+This could be so much more.
```

Comment:
What do you mean with `This` in this sentence?
```suggestion
This could be so much more.
```

Reply:
I wanted to express that we already saw that we can do more, that we want make this better, that ItchySats could be so much more (...). But I'm not sure this sentence adds much value. I'll just remove it :)